public/index.html

```diff
@@ -454,3 +456,545 @@ <h1 class='steps'>Team Blue Steel</h1>
 		</div>
 	</body>
 </html>
+=======
+=======
+>>>>>>> vlope093/master
```

Comment:
this are not suppose to be part of the code. Please double check before submitting a pull request